app/calendar_grid.py

```diff
@@ -0,0 +1,51 @@
+import calendar
+import datetime
+from typing import List, Tuple
+
+DAYS_OF_THE_WEEK: List[str] = ['Monday', 'Tuesday', 'Wednesday',
+                               'Thursday', 'Friday', 'Saturday', 'Sunday']
```

Comment:
This both hardcodes the language and the order of the dates which it should not.
calendar.day_name - gives you a list of week names in the requested locale.
calendar.setfirstweekday(calendar.SUNDAY) - sets the week to start on a Sunday, which is how it starts in Israel.

Use these instead.

You can change the locale used for the names with locale.setlocale(locale.LC_TIME, ("fr")). 
Hebrew should work with locale.setlocale(locale.LC_TIME, ("he", "UTF-8")) but couldn't get that to correctly show Hebrew text.

Reply:
True, its very important issue. 
For now I made it as a temporary var for displaying the overall layout,
There is an open ticket deals with this and its suppose to change it according the location. 
If it will not be done soon, maybe Ill do it by my self.

Reply:
That was really helpful.